spec/requests/phases_spec.rb

```diff
@@ -81,4 +81,111 @@
       end
     end
   end
+
+  describe 'GET /phases/:id/submissions' do
+    let(:challenge_user) { create_user(role: "challenge_manager") }
+    let(:challenge) { create(:challenge, user: challenge_user) }
+    let(:phase) { create(:phase, challenge: challenge) }
+    let!(:submission1) { create(:submission, challenge: challenge, phase: phase, status: 'submitted') }
+    let!(:submission2) { create(:submission, challenge: challenge, phase: phase, status: 'submitted') }
+    let!(:submission3) { create(:submission, challenge: challenge, phase: phase, status: 'submitted', judging_status: 'selected') }
+    let!(:submission4) { create(:submission, challenge: challenge, phase: phase, status: 'submitted', judging_status: 'winner') }
+
+    before { log_in_user(challenge_user) }
+
+    it 'assigns submissions and calculates status counts' do
```

Comment:
there is a principle in testing that you test the boundaries of the object and treat the internals as a black box. this spec tests the implementation details of the controller by asserting which instance variables are assigned. with higher level specs like a request spec, you really want to assert that the response body (output) is correct based on the request data (input), not the internals of how the server gets there while processing the request.

I would probably replace this spec with one that asserts the `submissions_by_status` counts are displayed to the user in the response body. if you wanted you could assert that all submissions are visible (none are filtered).